torch/csrc/autograd/engine.cpp

```diff
@@ -194,12 +189,12 @@ auto ReadyQueue::push(FunctionTask item) -> void {
 auto ReadyQueue::pop() -> FunctionTask {
   std::unique_lock<std::mutex> lock(mutex);
   not_empty.wait(lock, [this]{ return !heap.empty(); });
+  // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
```

Comment:
Is there a good way of keeping track of these NOLINTs aside from something like grep?

Reply:
clang-tidy does report how many `NOLINT`s it ran into at the end of a run, so we could use that. Otherwise yes grep